packages/eslint-plugin/docs/rules/no-explicit-any.mdx

```diff
@@ -145,6 +145,36 @@ interface Garply {
 }
 ```
 
+## Alternatives to `any`
+
+Which type(s) to use instead of `any` depends on the context.
+
```

Comment:
TODO: Add more straightforward cases from earlier in the file...

- If the type is known, describing it in an `interface` or `type`
  - Maybe mention `[i: string]`/etc. index signatures?
- If the type is not known, using the safer `unknown` type